src/jit/assertionprop.cpp

```diff
@@ -68,8 +65,10 @@ void Compiler::optAddCopies()
         return;
     }
 
-    for (lclNum = 0, varDsc = lvaTable; lclNum < lvaCount; lclNum++, varDsc++)
+    for (unsigned lclNum = 0; lclNum < lvaCount; lclNum++)
     {
+        LclVarDsc* varDsc = &lvaTable[lclNum];
```

Comment:
Should perhaps `operator[]` return `LclVarDsc*` rather than `LclVarDsc&` because most of the JIT code uses pointers to lclvars?